dp/tools/deployment/vendor/postgresql.yml

```diff
@@ -15,7 +15,8 @@ spec:
     spec:
       containers:
         - name: postgres
-          image: postgres:13.3
+          image: dp-postgres:latest
```

Comment:
where is this image generated and tagged? I do not find the string `dp-postgres` anywhere else in this changeset. Is there something missing that builds `dp/cloud/docker/postgresql/Dockerfile`?

Reply:
Indeed, I missed it, I just added  it to `skaffold.yml` on build stage